perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -28,9 +28,10 @@ olcAttributeTypes: {23}( 1.3.6.1.4.1.8057.2.80.27 NAME 'OIDCClientID' DESC 'Clie
 olcAttributeTypes: {24}( 1.3.6.1.4.1.8057.2.80.28 NAME 'institutionsCountries' DESC 'Countries of schacOrgs' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
 olcAttributeTypes: {25}( 1.3.6.1.4.1.8057.2.80.29 NAME 'isCesnetEligible' DESC 'isCesnetEligible' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
 olcAttributeTypes: {26}( 1.3.6.1.4.1.8057.2.80.30 NAME 'loa' DESC 'Level of authority' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {27}( 1.3.6.1.4.1.8057.2.80.31 NAME 'InternalUserIdentifiers' DESC 'All identifiers collected from UserExtSource attribute additionalIdentifiers' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```

Comment:
Please use camel-case, where starting letter is in lower-case - 'internalUserIdentifiers'.